Preparing your first release (#3255)
codemagic-ci-cd/codemagic-docs: content/yaml-publishing/google-play.md

12. Finally, click **Invite user** to finish setting up the service account on Google Play. In the Invite user window, the Email address field is pre-filled. Under Permissions, the default ones are already selected. You can go with these. Click Invite user at the bottom of the page.<br><br>
    ![Google play all](../uploads/s5.png)

## Preparing your first release
⚠️ Important: The very first version of your app must be uploaded to Google Play manually.

> 1. Run your Codemagic workflow to produce an .aab (look under Artifacts in your build logs).

Reviewer: would be good to add a link to how a AAB can be produced.

> 1. Run your Codemagic workflow to produce an .aab (look under Artifacts in your build logs).

Reviewer: good to have consistent names with AAB, so either .aab or app-release.aab

> 2. Download the generated app-release.aab.

Reviewer: ```suggestion
2. Download the generated `app-release.aab` binary from the build artifacts section.
```

3. Go to your Google Play Console, create your application entry (if it doesn’t exist yet), and manually upload this .aab in Production ![google_play_create_app](../uploads/2024/9/google_play_create_app.png)

4. Create new release (or the track you want to start with).
![google_play_create_new_release](../uploads/2024/9/google_play_create_new_release.png)

5. Complete the required app setup steps (app details, screenshots, content rating, etc.) and roll out your release.

After this first manual upload, Codemagic can automatically publish subsequent versions to the track(s) you configured in your codemagic.yaml.

> Also remember: each new build uploaded to Google Play must have a higher version code. You can automate this by [incrementing the build versions](../building/build-versioning/ 'Build versioning') on Codemagic.

Reviewer: ```suggestion
Also remember: each new build uploaded to Google Play must have a higher version code than the previously uploaded one. You can automate [incrementing the build versions](../building/build-versioning/ 'Build versioning') in your Codemagic workflows.
```
